sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/TrackNotImplementedException.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Tests.Diagnostics
```

Comment:
Let's add more test cases:
1. Create a type inheriting from NotImplementedException. Should we report on that?
2. Create an exception, but do not throw.